### DISCUSSION TITLE
Welcome to coding_playlists Discussions!

### DISCUSSION THREAD

**Discussion by gracekishino** [gracekishino/coding_playlists#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jainil2004** [comment]
hey @gracekishino, so i was thinking that we can improve the design of the 'contribute' link in the header tag. the original design is good and it does work but with the improved Spotify, Youtube section it just feels a little out of the place. And hence I was thinking that we add some sort of a smaller and more minimalistic link for contribution. just so that it fits well with other elements of the page and compliments other features of the page. Do let me know what do you think about this idea. I will create a issue regarding the re-design once you give me a go ahead. Thanks

**Comment by gracekishino** [comment]
@Jainil2004 thanks for bringing this up! It sounds like a good idea. I also thought the header looks slightly out of place with the footer having a slightly different design (Header has black background with white border at the bottom and footer has grey background and no border. 

So you can just update the 'contribute' link or also the header and/or footer if you like.

If it's easier, you can just create a pull request with the changes and I can check the preview from there.

**Reply by Jainil2004** [reply]
@gracekishino sure I would be very happy to help and improve. I'll see what I can do and what looks the best while still providing functionality. Give me some time and I'll reach back to you
Thanks
